sysconfig/sysconfig.go

```diff
@@ -53,15 +54,42 @@ type Options struct {
 	GadgetSnap snap.Container
 }
 
-// FilesystemOnlyApplyOptions is the set of options for
-// ApplyFilesystemOnlyDefaults.
-type FilesystemOnlyApplyOptions struct {
-	// Classic is true when the system in rootdir is a classic system
-	Classic bool
+// Device carries information about the device model and mode that is
+// relevant to sysconfig.
+type Device interface {
+	RunMode() bool
+	Classic() bool
+
+	Kernel() string
+	//Base() string
+
+	HasModeenv() bool
+
+	//Model() *asserts.Model
+}
+
+type deviceInfo struct {
+	model *asserts.Model
+}
+
+func (di *deviceInfo) RunMode() bool {
+	return false
```

Comment:
(nipick) maybe a small comment here that this is false because in the sysconfig context where this is used we are never in RunMode ?